Xamarin.Forms.Core/Entry.cs

```diff
@@ -26,6 +26,8 @@ public class Entry : InputView, IFontElement, ITextElement, ITextAlignmentElemen
 
 		public static readonly BindableProperty FontAttributesProperty = FontElement.FontAttributesProperty;
 
+		public static readonly BindableProperty IsTextPredictionEnabledProperty = BindableProperty.Create("IsTextPredictionEnabled", typeof(bool), typeof(Entry), true);
```

Comment:
can use `nameof(IsTextPredictionEnabled)` here

Reply:
Also set default binding mode to OneTime